File: app/src/test/java/com/tomaszrykala/dorefindmi/things/controller/digidisplay/NonBlockingDigiDisplayController.java
package com.tomaszrykala.dorefindmi.things.controller.digidisplay;

import com.tomaszrykala.dorefindmi.things.supplier.digidisplay.DigiDisplaySupplier;

public class NonBlockingDigiDisplayController extends DigiDisplayController {

    public NonBlockingDigiDisplayController(DigiDisplaySupplier digiDisplaySupplier) {
        super(digiDisplaySupplier);
    }

    @Override public void displayBlocking(String text, int millis) {
        super.displayBlocking(text, 0);
    }
}

File: app/src/test/java/com/tomaszrykala/dorefindmi/things/controller/digidisplay/NonBlockingDigiDisplayController.kt
package com.tomaszrykala.dorefindmi.things.controller.digidisplay

import com.tomaszrykala.dorefindmi.things.supplier.digidisplay.DigiDisplaySupplier

class NonBlockingDigiDisplayController(digiDisplaySupplier: DigiDisplaySupplier) : DigiDisplayController(digiDisplaySupplier) {

    override fun displayBlocking(text: String, millis: Int) {
        super.displayBlocking(text, 0)
    }
}
